exchanges/bybit/unmarshal.go

```diff
@@ -0,0 +1,10 @@
+package bybit
+
+import (
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+)
+
+// UnmarshalJSON implements the json.Unmarshaler interface for KlineItem
+func (k *KlineItem) UnmarshalJSON(data []byte) error {
+	return json.Unmarshal(data, &[7]any{&k.StartTime, &k.Open, &k.High, &k.Low, &k.Close, &k.TradeVolume, &k.Turnover})
+}
```

Comment:
🦍 Don't feel this is worth it's own file.
Only care because when I `/unmar` I'll get this.
Feel it could really live right next to the type, tbh.

Reply:
Got winged at by having funcs in types file, I prefer it hugged with the type as well. But I will be adding specific unmarshallers when #1957 gets merged so this will be more populated. So up to you.